native/SampleApps/RestAPIExplorer/RestAPIExplorer/AppDelegate.swift

```diff
@@ -139,6 +129,20 @@ class AppDelegate : UIResponder, UIApplicationDelegate
     }
     
     // MARK: - Private methods
+    func loginIfRequired() -> Void  {
```

Comment:
There is still a lot of boilerplate code (loginIfRequired, all the handle* methods). Could we provide that through a helper class and only require a one liner in the AppDelegate?

Reply:
The only code that are app specific is the setup of the root view controller, and the bit with the initial view controller. Could we move everything to a helper (loginIfRequired, handle logout and user switching)? Of course some apps might need more control and could look at the code of the helper as a guide, but all the sample apps / template apps would have much simple AppDelegate classes.